x-pack/metricbeat/module/aws/utils_test.go

```diff
@@ -377,7 +377,7 @@ func TestFindIdentifierFromARN(t *testing.T) {
 	}
 
 	for _, c := range cases {
-		identifier, err := FindIdentifierFromARN(c.resourceARN)
+		identifier, err := FindShortIdentifierFromARN(c.resourceARN)
```

Comment:
Can we also add tests for `FindWholeIdentifierFromARN` here please?
```suggestion
		shortIdentifier, err := FindShortIdentifierFromARN(c.resourceARN)
		assert.NoError(t, err)
		assert.Equal(t, c.expectedShortIdentifier, shortIdentifier)

		wholeIdentifier, err := FindWholeIdentifierFromARN(c.resourceARN)
		assert.NoError(t, err)
		assert.Equal(t, c.expectedWholeIdentifier, wholeIdentifier)
```

Reply:
yes, It is done

Reply:
Thank you!